pkg/reconciler/clusterducktype/clusterducktype_test.go

```diff
@@ -17,13 +17,19 @@ limitations under the License.
 package clusterducktype
 
 import (
+	"bufio"
 	"context"
 	"flag"
 	"fmt"
+	"io/ioutil"
 	"os"
+	"path"
 	"strings"
 	"testing"
 
+	"github.com/n3wscott/rigging/pkg/installer"
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+
```

Comment:
Those deserve to be grouped with the other third-party imports.

Reply:
yeahyeahyeah :D